test/e2e/common.ts

```diff
@@ -117,7 +117,7 @@ export const installAppAndDna = async (
     installed_app_id,
     agent_key: agent,
     path,
-    membrane_proofs: {},
+    roles_settings: {},
```

Comment:
`roles_settings` is optional
```suggestion
    roles_settings: {},
```

Reply:
Delete me

Reply:
done.